### PR TITLE
@types/googlemaps: Add 'className' to MarkerLabel interface

### PR DESCRIPTION
**Pull request by thepuzzlemaster**
This is my first time submitting a PR to this repo, so hopefully I'm doing this correctly.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://developers.google.com/maps/documentation/javascript/reference/marker#MarkerLabel.className>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header. (Header already indicates 3.4.3, which is where this prop was added)

### CODE REVIEW

File: types/googlemaps/reference/marker.d.ts [hunk 1]
```diff
@@ -458,6 +458,17 @@ declare namespace google.maps {
      * @see {@link https://developers.google.com/maps/documentation/javascript/reference/marker#MarkerLabel Maps JavaScript API}
      */
     interface MarkerLabel {
+        /**
+         * The className property of the label's element (equivalent to the element's class attribute).
+         * Multiple space-separated CSS classes can be added. Default is no CSS class (an empty string).
+         * The font color, size, weight, and family can only be set via the other properties of MarkerLabel.
```
Comment (demensky) [inline types/googlemaps/reference/marker.d.ts:464]: I suggest you mark the mention of `MarkerLabel` as a link to it.

```suggestion
         * The font color, size, weight, and family can only be set via the other properties of {@link MarkerLabel}.
```

File: types/googlemaps/reference/marker.d.ts [hunk 2]
```diff
@@ -458,6 +458,17 @@ declare namespace google.maps {
...
+         * CSS classes should not be used to change the position nor
+         * orientation of the label (e.g. using translations and rotations) if also using marker collision management.
+         * @default ''
+         * @see {@link https://developers.google.com/maps/documentation/javascript/reference/marker#MarkerLabel.className Maps JavaScript API}
+         * @see {@link https://developers.google.com/maps/documentation/javascript/examples/marker-collision-management marker collision management}
```
Comment (demensky) [inline types/googlemaps/reference/marker.d.ts:469]: How do you look at leaving the link where it was in the documentation?

```suggestion
         * CSS classes should not be used to change the position nor orientation of the label (e.g. using translations
         * and rotations) if also using marker {@link https://developers.google.com/maps/documentation/javascript/examples/marker-collision-management marker collision management}.
         * @default ''
         * @see {@link https://developers.google.com/maps/documentation/javascript/reference/marker#MarkerLabel.className Maps JavaScript API}
```

Comment (thepuzzlemaster) [reply types/googlemaps/reference/marker.d.ts]: Sounds good!

**Review by demensky — APPROVED** [review]

**Comment by thepuzzlemaster** [comment]
Ready to merge

**Pull request merged**